exporter/fileexporter/exporter.go

```diff
@@ -94,8 +117,31 @@ func (f *Exporter) Export(_ context.Context, _ *log.Logger, featureEvents []expo
 	return nil
 }
 
-// IsBulk return false if we should directly send the data as soon as it is produce
-// and true if we collect the data to send them in bulk.
-func (f *Exporter) IsBulk() bool {
-	return true
+func (f *Exporter) writeParquet(filePath string, featureEvents []exporter.FeatureEvent) error {
+	fw, err := local.NewLocalFileWriter(filePath)
+	if err != nil {
+		return err
+	}
+	defer fw.Close()
+
+	pw, err := writer.NewParquetWriter(fw, new(exporter.FeatureEvent), int64(runtime.NumCPU()))
+	if err != nil {
+		return err
+	}
```

Comment:
Do you know if it is possible to write some tests to check the errors?

Reply:
seems not easy,  one way is modifying `obj interface{}` in `writer.NewParquetWriter` using invalid parquet struct tag
but it will add one more param in `writeParquet` and the func will look less simple